jwst/assign_wcs/tests/test_nirspec.py

```diff
@@ -57,7 +57,7 @@ def create_hdul():
     phdu.header['instrume'] = 'NIRSPEC'
     phdu.header['detector'] = 'NRS1'
     phdu.header['time-obs'] = '8:59:37'
-    phdu.header['date-obs'] = '2014-09-05'
+    phdu.header['date-obs'] = '2016-09-05'
```

Comment:
But isn't 2016-09-05 already later than ~Nov. 2015 and hence will always get CV3-era ref files selected, because the ref files have USEAFTER < 2016.